spec/lib/video_info/providers/youtube_api_spec.rb

```diff
@@ -0,0 +1,392 @@
+require 'spec_helper'
+
+describe VideoInfo::Providers::Youtube do
+  before(:all) do
+    VideoInfo.provider_api_keys = { youtube: 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0' }
+  end
+
+  describe ".usable?" do
+    subject { VideoInfo::Providers::Youtube.usable?(url) }
+
+    context "with youtube.com url" do
+      let(:url) { 'http://www.youtube.com/watch?v=Xp6CXF' }
+      it { is_expected.to be_truthy }
+    end
+
+    context "with youtu.be url" do
+      let(:url) { 'http://youtu.be/JM9NgvjjVng' }
+      it { is_expected.to be_truthy }
+    end
+
+    context "with other url" do
+      let(:url) { 'http://google.com/video1' }
+      it { is_expected.to be_falsey }
+    end
+
+    context "with playlist url" do
+      let(:url) { 'http://www.youtube.com/playlist?p=PLA575C81A1FBC04CF' }
+      it { is_expected.to be_falsey }
+    end
+  end
+
+  describe "#available?" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.